### DISCUSSION TITLE
[REQUEST] GetListingOffers needs to return Minimum Order Quantity

### DISCUSSION THREAD

**Discussion by Domino-the-cat** [amzn/selling-partner-api-models#586] [Ideas]
Hi

We are calling 

GET /products/pricing/v0/listings/{SellerSKU}/offers

and it is returning all the documented data BUT to then use this data to accurately reprice our offers it would be super helpful if it also returned Minimum Order Quantity for each offer.

For example we called this against a listing and it showed a competitor was selling 20% below our offer. When we looked on the item web page we found the cheaper seller had added an MOQ.

Without MOQ in the /products/pricing/v0/listings/{SellerSKU}/offers this function is of limited use

Please could this be considered for development

Thanks

Dom

**Comment by Domino-the-cat** [comment]
Any thoughts anyone?

thanks Dom

**Comment by Domino-the-cat** [comment]
I posted a case with developer support on this issue, unfortunately they seem unable to understand and just keep replying with the same generic reply which has no connection to do with pricing, is there anyway to speak to someone?

this was the reply>>>>

Greetings from Amazon,

My name is xxxxxxxx, from the Developer Support team

The Selling Partner API for Orders helps you programmatically retrieve order information.

These APIs let you develop fast, flexible, custom applications in areas like order synchronization, order research, and demand-based decision support tools.

To know more about the Orders API v0 reference:
https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#order

To know more about the Orders API v0 model:
https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-model

Please let us know how we did.

Were you satisfied with the support provided?

**Comment by 0xs3n4** [comment]
I hope you have solved your problem, I also have an impediment related to this... I would like the listing information from an ASIN, do you know if it is possible? from an ASIN to have access to the number of FMB/FBA sellers and the value of each offer?

**Reply by 0xs3n4** [reply]
I was passing by to update myself and saw this message, I managed to solve this problem by accessing this endpoint -> [endpoint](https://developer-docs.amazon.com/sp-api/docs/product-pricing-api-v0-reference#getitemoffers)
